GoogleForCreators/web-stories-wp
# Hide canvas overflow when trimming a video

## Context

This PR allows conditional hiding of overflow in the edit layer.

Regular media edit will not hide overflow (to allow you to see the full image when editing the zoom of e.g. the background image).

Video trimming however will always hide overflow, to actually only display the part of the video, that is visible and also not allow a video to cover the trim rail and functionality therein.

## User-facing changes

![hide-trim-overflow](https://user-images.githubusercontent.com/637548/135149750-ddc1e501-1b97-4706-b35b-eba751625b8c.gif)

## Testing Instructions

This PR can be tested by following these steps:

1. Add a video partially off-canvas
2. Double-click to edit the video zoom
3. Observe that the off-canvas part of the video is visible
4. Exit edit mode
5. Enter video trim mode (through quick-actions or design panel)
6. Observe that the off-canvas part of the video is *not* visible

## Checklist




---

Fixes #9181
